src/helpers.js

```diff
@@ -48,16 +49,43 @@ export function fileDownload ({ data, fileName = 'myFile.dat', mimeType = 'appli
     if (navigator.msSaveBlob) { // IE10
       return navigator.msSaveBlob(new Blob([data], { type: mimeType }), fileName)
     } else if ('download' in a) { // html5 A[download]
-      a.href = `data:${mimeType},${encodeURIComponent(data)}`
+      const myForm = document.createElement('form')
+      const textArea1 = document.createElement('textarea')
```

Comment:
I know about using input elements, but it didn't work, I'm not sure why. But I can give it one more try.

Reply:
Don't spend too much time on it.  If it didn't work as input fields, maybe there is a default size limit difference.  Sticking with textareas because of that makes sense.  Maybe a short code comment "textarea used to avoid text size limit issues" will help prevent a future intern from wasting time refactoring code...